composer.json

```diff
@@ -16,6 +16,7 @@
         "drupal/lb_accordion" : "^2.1.0",
         "drupal/lb_cards" : "^2.0.4",
         "drupal/lb_carousel" : "^2.0.3",
+        "drupal/lb_copy_section" : "^1.0",
```

Comment:
I agree with @podarok. If we're not installing and configuring this module, then we should not include it in composer and simply document the installation and use. I'm checking in with Shelley to discuss the best path forward.

Reply:
@rollki I've discussed with Shelley and also tested the module and I think it's ok to go ahead with, but we need to test and ensure it works well before committing. 

I tested and it seems there are some issues with cloning. My test steps:
- go to the homepage
- clone the body section
- edit the cloned block
- BUG 1: Nested entity references are not editable.
- BUG 2: Creating a new sub-block and attempting to save results in the error "This entity could not be referenced"
  - I ran into this on another project and had to add two different patches and make some changes in how entity references work. It will probably take more work to make complete. https://www.drupal.org/node/3098245 https://www.drupal.org/project/inline_entity_form/issues/3002175
- BUG 3: "Copy" text is larger than "Configure"

<img width="961" alt="SCR-20240423-jpwe" src="https://github.com/YCloudYUSA/y_lb/assets/238201/36645e1c-0394-4ed6-b865-8047f569403e">

<img width="1415" alt="SCR-20240423-joub" src="https://github.com/YCloudYUSA/y_lb/assets/238201/22b70991-74c9-4994-a2f3-ad1cf25b6ee5">

<img width="476" alt="SCR-20240423-jqom" src="https://github.com/YCloudYUSA/y_lb/assets/238201/645ae846-249d-42b2-aa6f-f235d083c7a7">

Reply:
@froboy, BUG 1 and BUG 2 seem unrelated to the Clone section module functionality. I think it can related to the way of demo content creation (migrations). If you create a new LP with LB and check the "copy/past section" functionality there - everything works fine for me.
![image](https://github.com/YCloudYUSA/y_lb/assets/744406/f58958d8-c8e7-442d-b566-73ea1e77f020)
cc @rollki

Reply:
@aleevas thanks. I think many users reuse the content in the demo pages, so I'd prefer to resolve the issue with demo pages before we move this forward.

My test case:

- Edit layout on an existing demo content page (e.g. Home page)
- Copy section with Cards (e.g. Home page body)
- Edit the cloned Cards block
- Add a new card
- Attempt to save the block
- Expected behavior: block saves
- Observed behavior: "Entity can not be referenced" error. Block can not be saved.

Also, if we're installing it we do need to enable for new installs and updates.

Reply:
the issue with cloned blocks should be fixed in this PR https://github.com/YCloudYUSA/y_lb_demo_content/pull/45

Reply:
Ok, I think this solves the current problems even though it affects a few modules. There is probably no simple solution.

But now since we make blocks for LB reusable at https://github.com/YCloudYUSA/y_lb_demo_content/pull/45 it means that these cloned blocks will appear in the "All system blocks" section when adding a block to LB, there mainly system blocks and probably few people use these blocks, but here is the question: will it be ok for us?

Reply:
@rollki I _strongly disagree_ with this approach.

as you say:

> it means that these cloned blocks will appear in the "All system blocks" section when adding a block to LB...

This means that dozens or hundreds of blocks will be listed in "All system blocks". This is a very bad idea and will cause chaos for content editors.

Reply:
See my comment above:

> I ran into this on another project and had to add two different patches and make some changes in how entity references work. It will probably take more work to make complete. 
> https://www.drupal.org/node/3098245 
> https://www.drupal.org/project/inline_entity_form/issues/3002175

We'll probably need some patches to get this to work (which will resolve other issues related to cloning) if we still want to proceed down this path. I'm not sure though.